flask_sqlalchemy/utils.py

```diff
@@ -0,0 +1,20 @@
+
+from pkg_resources import parse_version
```

Comment:
Don't use `pkg_resources` here, it forces a dependency on setuptools. Use [`packaging.version.parse`](https://packaging.pypa.io/en/latest/version/#packaging.version.parse) if you really need to, but I'd rather not add a new dependency. For what's needed here, you can probably get away with `tuple(int(x) for x in sqlalchemy.__version__.split(".")[:2]) < (1, 3)`.

Reply:
Don't need this anymore since the above change request removed the only usage of needing to know the SA version.

Reply:
oops, nm, need that in the tests.

Reply:
k, resolved locally with no dependency.